[DBS-002] Set web frontend settings
--- zabbix-4.4.5-orig/frontends/php/include/defines.inc.php	2019-08-27 08:12:50.440205900 +0300
+++ zabbix-4.4.5/frontends/php/include/defines.inc.php	2019-08-27 08:14:09.632876497 +0300
@@ -1307,15 +1307,15 @@
 // input fields
 define('ZBX_TEXTAREA_HTTP_PAIR_NAME_WIDTH',		218);
 define('ZBX_TEXTAREA_HTTP_PAIR_VALUE_WIDTH',	218);
-define('ZBX_TEXTAREA_MACRO_WIDTH',				250);
-define('ZBX_TEXTAREA_MACRO_VALUE_WIDTH',		300);
+define('ZBX_TEXTAREA_MACRO_WIDTH',				400);
+define('ZBX_TEXTAREA_MACRO_VALUE_WIDTH',		400);
 define('ZBX_TEXTAREA_TAG_WIDTH',				250);
 define('ZBX_TEXTAREA_TAG_VALUE_WIDTH',			300);
 define('ZBX_TEXTAREA_COLOR_WIDTH',				96);
 define('ZBX_TEXTAREA_FILTER_SMALL_WIDTH',		150);
 define('ZBX_TEXTAREA_FILTER_STANDARD_WIDTH',	300);
 define('ZBX_TEXTAREA_TINY_WIDTH',				75);
-define('ZBX_TEXTAREA_SMALL_WIDTH',				150);
+define('ZBX_TEXTAREA_SMALL_WIDTH',				450);
 define('ZBX_TEXTAREA_MEDIUM_WIDTH',				270);
 define('ZBX_TEXTAREA_STANDARD_WIDTH',			453);
 define('ZBX_TEXTAREA_BIG_WIDTH',				540);
@@ -1429,7 +1429,7 @@
 // the default theme
 define('ZBX_DEFAULT_THEME', 'blue-theme');
 
-define('ZBX_DEFAULT_URL', 'zabbix.php?action=dashboard.view');
+define('ZBX_DEFAULT_URL', 'zabbix.php?action=problem.view');
 
 // date format context, usable for translators
 define('DATE_FORMAT_CONTEXT', 'Date format (see http://php.net/date)');
